api/config.json

```diff
@@ -49,7 +49,8 @@
     {
       "name": "Hyperlane",
       "tags": [
-        "Hyperlane"
+        "Hyperlane",
+        "Hyperlane Warp"
```

Comment:
I know I made this change, but wondering if this should just be "Warp"?

Reply:
Warp is more succinct and unless we ever plan on adding another module with the same name i don't see why it wouldn't be sufficiently clear. I'm mostly indifferent on this detail though!